### DISCUSSION TITLE
最新版本没法运行

### DISCUSSION THREAD

**Discussion by hckhor88** [jynew/jynew#956] [Q&A]
很多报错, 请问如何解决，谢谢

我用的是版主的版本，我没改过任何东西,
尝试使用两个版本也无法运行
2020.3.25f1c1
2020.3.23f1c1

![image](https://user-images.githubusercontent.com/103172707/162223640-ad13bafe-5cdf-43e8-9300-c4cc4e3570ee.png)

**Comment by chenggongchina** [comment]
应该是升级unity包的时候失败了，更新到最新版，删除packages和libirary文件夹然后重试下。
